packages/app/src/pages/PatientPostCall.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { CustomContainer } from '../components';
+import { useParticipant } from '../store';
+
+// TODO: Merge 2 post call pages in one
+export const PatientPostCall = (): JSX.Element => {
+  const { providerName } = useParticipant();
+  const { t } = useTranslation();
+
+  // TODO hard-coded data #Post-Call ticket
+  const mockCallDuration = '15:05';
```

Comment:
Is it tough to make this not mock data?

Reply:
yes we should calculate this from encounter resource time.
so couple options here: the simplest  calculate directly from `encounter.period.start` which is start time of patients call in ISOformat, but the call duration won't be accurate as patient can wait for some time. another option maybe store time when provider joins call in `context `etc. for both patient and provider and show that?

Reply:
For now it seems better to go with `encounter.period.start` to keep the implementation using our FHIR resources

Reply:
@aykhanahmadli I believe you resolved this comment thread, correct?